src/pages/InteractiveAPI/InteractiveAPI.vue

```diff
@@ -103,6 +103,9 @@ query { hello }
       const queryObject = gql`
         ${query}
       `
+
+      if (queryObject.definitions?.[0]?.operation == 'mutation') return query
```

Comment:
Just a few lines down there's `// Exclude queries that do not support limiting` which makes this seem like it's not exiting early because it's excluded. Perhaps a comment above this line would be nice too? `// Don't limit mutations because...`

Reply:
Sure, I've added a comment for that 👍